docs/resources/hss_host_protection.md

```diff
@@ -0,0 +1,105 @@
+---
+subcategory: "Host Security Service (HSS)"
+---
+
+# huaweicloud_hss_host_protection
+
+Manages an HSS host protection resource within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable "host_id" {}
+variable "version" {}
+variable "charging_mode" {}
+
+resource "huaweicloud_hss_host_protection" "test" {
+  host_id       = var.host_id
+  version       = var.version
+  charging_mode = var.charging_mode
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region to which the HSS host protection resource belongs.
+  If omitted, the provider-level region will be used. Changing this parameter will create a new resource.
+
+* `host_id` - (Required, String, ForceNew) Specifies the host ID for the host protection.
+  Changing this parameter will create a new resource.
+
+  -> Before using host protection, it is necessary to ensure that the agent status of the host is **online**.
+
+* `version` - (Required, String) Specifies the protection version enabled by the host.  
+  The valid values are as follows:
+  + **hss.version.basic**: Basic version.
+  + **hss.version.advanced**: Professional version.
+  + **hss.version.enterprise**: Enterprise version.
+  + **hss.version.premium**: Ultimate version.
+
+* `charging_mode` - (Required, String) Specifies the charging mode for host protection.  
+  The valid values are as follows:
+  + **prePaid**: The yearly/monthly billing mode.
+  + **postPaid**: The pay-per-use billing mode.
+
+* `quota_id` - (Optional, String) Specifies quota ID for host protection.
+  If omitted, randomly select quota for the corresponding version.
```

Comment:
Does this parameter need to be explained: `This field is valid only when `charging_mode` is set to **prePaid**`.

Reply:
fixed.